### PR TITLE
multisig support tdpos

### LINKED ISSUE #158: Multisig supports tdpos proposals and votes
**Brief of your idea**
Multisig supports tdpos proposals and votes

**What could be better if XuperChain using your idea**
tdpos can be work by more than one person

**Are there any side effects?**

**Additional information**

### PR DESCRIPTION
**Pull request by MichelleZ**
## Description

Fixes #158 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Brief of your solution
Multisig scene supports tdpos

## How Has This Been Tested?
You can use multisig cmd tool to operate this function

### CODE REVIEW

File: cmd/cli/comm_trans.go [hunk 1]
```diff
@@ -97,6 +97,9 @@ func (c *CommTrans) GenPreExeRes(ctx context.Context) (
 		if tmpReq != nil {
 			preExeReqs = append(preExeReqs, tmpReq)
 		}
+		if tmpReq == nil && err == nil {
```
Comment (yucaowang) [inline cmd/cli/comm_trans.go]: use "else if" may reduce a tmpReq value comparing?

File: cmd/cli/comm_trans.go [hunk 2]
```diff
@@ -139,6 +142,10 @@ func (c *CommTrans) GetInvokeRequestFromDesc() (*pb.InvokeRequest, error) {
 	}
 
 	var preExeReq *pb.InvokeRequest
+	if bytes.Contains(desc, []byte("tdpos")) {
```
Comment (yucaowang) [inline cmd/cli/comm_trans.go]: so I can write a description contains "tdpos" for any transaction to enforce using multisign?

File: cmd/cli/comm_trans.go (hunk 1 above)
Comment (MichelleZ) [reply cmd/cli/comm_trans.go]: Fix it

File: cmd/cli/comm_trans.go (hunk 2 above)
Comment (MichelleZ) [reply cmd/cli/comm_trans.go]: So replace tdpos with contract_name, it's more general

**Pull request merged**